packages/common/src/store/pages/chat/sagas.ts

```diff
@@ -432,6 +433,12 @@ function* watchAddMessage() {
   yield takeEvery(addMessage, ({ payload }) => fetchChatIfNecessary(payload))
 }
 
+function* watchFetchChatRecheckPermissions() {
```

Comment:
I can just name it `fetchChat`, but I was worried that if we didn't make it super clear that one version was checking for recheck permissions and busting cache vs `fetchChatIfNecessary`, one might call this function thinking it would bust cache, but instead it would just return the existing chat.

Reply:
Ahh, I see what you're saying. Maybe cleanest would just be to have a `watchFetchChatIfNecessary` action which itself has a `bustCache` parameter (default false)? Nice to keep it 1:1 with action + saga if possible